src/swagger-specs/segmentation.yaml

```diff
@@ -3150,89 +3135,119 @@ components:
               "$ref":
                 type: string
                 example: https://ns.adobe.com/xdm/context/profile__union
+          description: This value is only used on Platform-generated audiences.
```

Comment:
Again, hard to tell what property this is describing when we put if after all the object's sub-properties. This description doesn't tell me anything about what this property is either -- only what kinds of audiences use it.